WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -253,6 +253,15 @@ private extension ProductsViewController {
         }
         coordinatingController.start()
     }
+
+    func clearFilter(sourceBarButtonItem: UIBarButtonItem? = nil, sourceView: UIView? = nil) {
```

Comment:
❓ I added this next to `addProduct` action. However, it seems it's located in `Navigation Bar Actions` extension even though it's technically not a navigation bar action. Is this fine or should we move it somewhere else?

Reply:
There is a extension for Actions, that encapsulates, for example, `filterButtonTapped`. Moving this method to that particular extension might be slightly more cohesive, but to be honest, I think either would work.

Reply:
Yeah that feels better, fixed in https://github.com/woocommerce/woocommerce-ios/pull/5372/commits/cbd9a6cdd0f0920421ad321ea37b141391e46d1c.